logserver/pom.xml

```diff
@@ -0,0 +1,72 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>alluxio-parent</artifactId>
+        <groupId>org.alluxio</groupId>
+        <version>1.6.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>alluxio-logserver</artifactId>
+    <packaging>jar</packaging>
+    <name>Alluxio Log Server</name>
+    <description>A logging server receiving log messages from Alluxio servers</description>
+
+    <properties>
+        <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+        <!-- run properly from sub-project directories -->
+        <license.header.path>${project.parent.basedir}/build/license/</license.header.path>
+        <checkstyle.path>${project.parent.basedir}/build/checkstyle/</checkstyle.path>
+        <findbugs.path>${project.parent.basedir}/build/findbugs/</findbugs.path>
+        <failIfNoTests>false</failIfNoTests>
+    </properties>
+
+    <dependencies>
+        <!-- External dependencies -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.16</version>
```

Comment:
The versions should already be handled by dependency management. However, it looks like our version string and dependency management are out of sync. @apc999

Reply:
@riversand963 Could you modify the parent pom.xml to use `log4j.version` in the dependency management section instead of the hardcoded `1.2.16` (we want to use `1.2.17` which is the version defined by the variable)?

Reply:
Done.

Reply:
The parent pom will be pushed soon.